src/doc/contrib/src/issues.md

```diff
@@ -217,17 +221,17 @@ There are several things to consider when triaging an issue:
 * Assuming the issue looks valid, remove the [S-triage] label and move it onto
```

Comment:
> remove the [S-triage] label

This is part of what doesn't sit well with me. What if the next natural status is one of the **(reserved for Cargo team)** ones? Isn't removing [S-triage] reserved as well then?

I suggest something like this:

> Assuming the issue looks valid, consider assigning it a new status label:
>
> ... (bullet points)
>
> Remove the [S-triage] label if you moved the issue to a new status.

Reply:
I tried a slightly different wording

> Assuming the issue looks valid, switch the [S-triage] label for one of the following:

My thinking is that this makes it clearer that this is an "atomic" operation; that you should only remove `S-triage` if you can add the new label.  In practice, if there is no more design work, then its "needs-team-input".

Reply:
This change looks reasonable. Thanks!